ads/_config.js

```diff
@@ -374,6 +374,11 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  innity: {
+    prefetch: 'https://cdn.innity.net/admanager.js',
```

Comment:
we have `renderStart` and `noContent` [API](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) that helps to improve user experience. Please consider implementing them.